src/components/TimeBasedLineChart.js

```diff
@@ -227,6 +226,7 @@ const TimeBasedLineChart = React.createClass({
       shadeBelowZero: false,
       shadeFutureOnGraph: true,
       showBreakPoint: true,
+      showZeroLine: false,
```

Comment:
This is a breaking change. Since we aren't providing any backwards compatibility, we need to make sure we bump the second number in the version.